src/main.rs

```diff
@@ -1027,7 +1022,7 @@ fn hackstead_blocks(
             .as_ref()
             .filter(|cert| {
                 if cert.requires_xp {
-                    tiles_owned < hs_adv_sum.land.try_into().unwrap()
+                    tiles_owned < hs_adv_sum.land as usize
```

Comment:
This still needs error handling, no?

Reply:
The original line fails to build with this error, and I can't figure out where to put the type annotations:
```
error[E0283]: type annotations needed
    --> src/main.rs:1025:33
     |
1025 |                     tiles_owned < hs_adv_sum.land.try_into().unwrap()
     |                                 ^ -------------------------- this method call resolves to `Result<T, <Self as std::convert::TryInto<T>>::Error>`
     |                                 |
     |                                 cannot infer type
     |
     = note: cannot satisfy `usize: PartialOrd<_>`
```